### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Gracetemitope**
In this PR, we implemented Unit testing using jests. We tested the following:

Add functionality
Delete Functionality
Local storage.

We also mocked the dom and local storage

### CODE REVIEW

File: src/__mocks__/taskWrk.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  });
+
+  test('Create DOM', () => {
+    document.body.innerHTML = '<ul id="container_ul"></ul>';
+    component();
+    const list = document.querySelectorAll('#container_ul li');
+    expect(list).toHaveLength(3);
+  });
+});
+
```
Comment (mimipeshy) [inline src/__mocks__/taskWrk.test.js]: - [x] Let's add more tests to cover negative scenarios as well. For example test for what happens a new object is not created? we want to the code to anticipate this before it happens

File: src/__mocks__/taskWrk.test.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+describe('Delete functions', () => {
+  test('Delete test', () => {
+    deleteTrash();
+    const trash = document.querySelector("li[id='1'] > div > i");
+    trash.click();
+    expect(tasks).toHaveLength(2);
+  });
+});
+
```
Comment (mimipeshy) [inline src/__mocks__/taskWrk.test.js]: - [x] Let's add more tests for this function as well. Test if it can delete one element to the DOM.

File: src/__mocks__/taskWrk.test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+describe('Storage test', () => {
+  test('Storage', () => {
+    storage();
+    const tasksOnStorage = JSON.parse(localStorage.getItem('tasks'));
+    console.log(tasksOnStorage);
```
Comment (mimipeshy) [inline src/__mocks__/taskWrk.test.js]: - [x] I think it's best if we remove the console.log statements in production. It's a good practice when in development mode because it helps trace bugs but in production it makes the code base look untidy.

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @Gracetemitope and @GuilhermeGiachellin  👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights

- good GitHub workflow ✔️ 
- test cases pass well ✔️ 

## Required Changes ♻️

- [x] In your README file add instructions on how to run tests

- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

### Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


### Happy coding!👏👏👏
